Makefile

```diff
@@ -325,10 +324,11 @@ beforedepend:: utils/config.ml
 # Start up the system from the distribution compiler
 .PHONY: coldstart
 coldstart:
+	echo $(CAMLC)
```

Comment:
Is this a leftover debug statement, or something you intended to keep in the PR?

Reply:
Leftover debug! Thanks for spotting.